bdd/spec/001_event_lnn_spec.sh

```diff
@@ -111,13 +111,13 @@ Describe "Events"
     INTOUTPUT="I got a number! 5"
 
     It "runs js"
-      Pending methods-interfaces-and-operators
+      Pending generics
```

Comment:
Why would this test still be pending? I don't see any generics in here?

Reply:
ah, i was thinking there was some math in the test for some strange reason. I'll enable the test